ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/LedgerDB.hs

```diff
@@ -9,10 +11,9 @@ import qualified Test.Ouroboros.Storage.LedgerDB.InMemory as InMemory
 import qualified Test.Ouroboros.Storage.LedgerDB.OnDisk as OnDisk
 
 tests :: TestTree
-tests = testGroup "LedgerDB" [
-      HD.tests
-    , InMemory.tests
-    , OnDisk.tests
-    , DiskPolicy.tests
-    , DbChangelog.tests
-    ]
+tests = testGroup "LedgerDB" ([ HD.tests | os /= "mingw32" ] <>
```

Comment:
Possibly add a FIXME for removing this at some point?

Reply:
Done